src/pages/Welcome/TeamSection/TeamSection.tsx

```diff
@@ -1,8 +1,10 @@
-import { UserOutlined, BankOutlined, EnvironmentOutlined, EditOutlined } from '@ant-design/icons';
-import { DEVELOPERS_INFO } from './TeamSectionConstants';
-import { Typography, Card, Flex, Divider } from 'antd';
-import styles from './TeamSection.module.scss';
+import GitHubLogo from '@/assets/svg/github.svg?react';
```

Comment:
You can use react-icons lib:
`import { SiGithub } from "react-icons/si";`

Reply:
ok